pom.xml

```diff
@@ -99,9 +109,9 @@
 
         <!-- Request/Response logging -->
         <dependency>
-            <groupId>net.rakugakibox.spring.boot</groupId>
+            <groupId>dev.akkinoc.spring.boot</groupId>
```

Comment:
Any particular reason we're using this group, and not the official spring one? https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter